apps/src/code-studio/initApp/loadApp.js

```diff
@@ -51,6 +52,17 @@ export function setupApp(appOptions) {
     }
   }
 
+  // if this level has a template level and we are in edit start blocks/sources mode,
+  // the workspace should be read only. The start blocks or sources for a level with a project
```

Comment:
> The start blocks or sources for a level with a project template are the start blocks or sources of the template.

This is only true if start blocks/sources are actually defined in the template. If these fields are blank in the template then the start blocks/sources defined in the non-template level are used.